proto/steward/v3/steward.proto

```diff
@@ -16,7 +16,7 @@ import "cellar_v2.proto";
 /*
  * Service for handling Cellar contract calls
  */
-service ContractCall {
+service ContractCallService {
```

Comment:
Renaming the service is a GRPC API-breaking change. We will need to either leave it the same or bump the major version of the API.